### PR TITLE
add back RN android packages

### PR DESCRIPTION
**Pull request by dulmandakh**
add back "add-ons;addon-google_apis-google-23" and "system-images;android-19;google_apis;armeabi-v7a" packages

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -64,5 +64,7 @@ RUN curl -sS https://dl.google.com/android/repository/${SDK_VERSION} -o /tmp/sdk
         "emulator" \
         "platforms;android-$ANDROID_BUILD_VERSION" \
         "build-tools;$ANDROID_TOOLS_VERSION" \
-        "system-images;android-19;default;armeabi-v7a" \
+        "add-ons;addon-google_apis-google-23" \
+        "system-images;android-19;google_apis;armeabi-v7a" \
+        "system-images;android-28;google_apis;x86_64" \
```
Comment (gengjiawen) [inline Dockerfile]: x86_64 arch will not run CI, you need arm arch.

Comment (dulmandakh) [reply Dockerfile]: During Google IO 2019, they say they added emulator-headless command to be used in CI

Comment (gengjiawen) [reply Dockerfile]: Can you try the `x86_64` can start on CircleCI, last time I tried, it won't work.

Comment (dulmandakh) [reply Dockerfile]: my internet connection is slow and it's taking so long to build Docker image on my machine.

Comment (gengjiawen) [reply Dockerfile]: You can use Circle CI debug build, in this way you can ssh to Circle CI machine for 2 hours.

Comment (dulmandakh) [reply Dockerfile]: took time and tested, it didn't work. fixed my mistake in this PR. Thank you

Comment (dulmandakh) [reply Dockerfile]: I thought that emulator-headless might work on CI, but still requires hardware acceleration.

**Pull request merged**